OpenSim/Simulation/Wrap/WrapObject.cpp

```diff
@@ -64,7 +64,8 @@ void WrapObject::constructProperties()
     constructProperty_quadrant("Unassigned");
     Appearance defaultAppearance;
     defaultAppearance.set_color(SimTK::Cyan);
-    defaultAppearance.set_representation(VisualRepresentation::DrawWireframe);
+    defaultAppearance.set_opacity(0.5);
+    defaultAppearance.set_representation(VisualRepresentation::DrawSurface);
```

Comment:
Could you explain the reason for changing this? Just curious. 

Does this change affect backwards compatibility?